### PR TITLE
2434 vtk poly data normals add compute cell normals

### LINKED ISSUE #2434: vtkPolyDataNormals - Cell Normals
### Need
At the moment, the `vtkPolyDataNormals` is only able to calculate the vertex normals.
I might be interesting to also compute the cell normals instead based on a programmatically set boolean.

Based on the suggestion of @finetjul in [this reply](https://discourse.vtk.org/t/polydata-normals/8550/2?u=emil-peter)

ps. You can assign me on this issue.

### PR DESCRIPTION
**Pull request by finetjul**
### Context
fix #2434 

### Results
Add `setComputePointNormals` and `setComputeCellNormals`

### Changes
- [x] Documentation and TypeScript definitions were updated to match those changes

### PR and Code Checklist
- [x] [semantic-release](https://github.com/semantic-release/semantic-release) commit messages
- [x] Run `npm run reformat` to have correctly formatted code

### Testing
- [x] This change adds or fixes unit tests 
- [x] Tested environment:
  - **vtk.js**: master
  - **OS**: Windows
  - **Browser**: Chrome

### CODE REVIEW

**Review by floryst — COMMENTED** [review]
LGTM overall.

File: Sources/Filters/Core/PolyDataNormals/test/testPolyDataNormals.js [hunk 1]
```diff
@@ -24,3 +28,70 @@ test('Test vtkPolyDataNormals passData', (t) => {
 
   t.end();
 });
+
+test.only('Test vtkPolyDataNormals normals', (t) => {
```
[the remark this answers is not in the document]
Comment (finetjul) [reply Sources/Filters/Core/PolyDataNormals/test/testPolyDataNormals.js]: fixed

File: Sources/Rendering/Core/Glyph3DMapper/index.d.ts [hunk 2]
```diff
@@ -87,6 +87,12 @@ export interface vtkGlyph3DMapper extends vtkMapper {
 	 */
 	getPrimitiveCount(): IPrimitiveCount;
 
+	/**
+	 * Sets the name of the array to use as orientation.
+	 * @param {String} arrayName Name of the array
+	 */
+	setOrientationArray(arrayName: Nullable<string>): boolean;
```
Comment (floryst) [inline Sources/Rendering/Core/Glyph3DMapper/index.d.ts:94]: `Nullable` is not imported

**Comment by finetjul** [comment]
I'll check the failing tests...

**Comment by floryst** [comment]
LGTM

**Pull request merged**